src/containers/OpenSeadragonViewer.js

```diff
@@ -58,6 +58,7 @@ const mapDispatchToProps = {
 const styles = {
   osdContainer: {
     flex: 1,
+    pointer: 'grab',
```

Comment:
do we want to use the 'cursor' selector?

Reply:
Whoops!
https://github.com/ProjectMirador/mirador/pull/3644/commits/4d79018b9434b78c3828248527f143277b986947

Reply:
navigation pane fixed with this commit:
https://github.com/ProjectMirador/mirador/pull/3644/commits/0fc4c3b2198c9ee987a474ae97fb150c2ae89915